app.R

```diff
@@ -2,7 +2,7 @@
 # To deploy, run: rsconnect::deployApp()
 # Or use the blue button on top of this file
 
-Sys.setenv(R_CONFIG_ACTIVE = "default") # Replace "default" with your config
+Sys.setenv(R_CONFIG_ACTIVE = "veoibd") # Replace "default" with your config
```

Comment:
```suggestion
Sys.setenv(R_CONFIG_ACTIVE = "default") # Replace "default" with your config
```
On GH, I'd like to keep this as `default`. When run it on the server, you would update this on your local version to be `veoibd`.